specifications/Staking Lockup.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Staking lockup to incentivize illiquidity
+author: @vtleonardo, @nelsonhp
+discussion: https://github.com/alicenet/alicenet/pull/340
+team: Backend
+category: Development
+status: Review
+related: -
+created: 2022-11-14
+---
+
+## Introduction
+
+### Summary
+
+This specification proposes a series of mechanisms to incentivize the illiquidity of ALCA token at the beginning of the AliceNet lifetime. The main component of this mechanism will be a Lockup smart contract that will allow users to lock their public staking positions in exchange for gaining extra rewards.
+
+### Context
+
+Once we release the Alicenet economics system, users will be able to get ALCA by migrating the old token (MadToken) directly into the ALCA smart contract. In addition to this, we also foresee the creation of AMM pools with ALCA in popular markets (UNISWAP V3, Balancer, etc), where users will be able to buy ALCA directly using different currencies.
+
+With ALCA tokens in hand, users will be able to stake into the Public Staking contract, and in the near future, they will be able to participate in the governance system to decide the future of the protocol.
+
+Since ALCA is the main currency of this project, the success of the ALCA is tightly tied to the success of the protocol. In the early life of the project, the ALCA price will be very susceptible to threats such as actors trying to dump/manipulate the price, massive exits by speculation, etc. Therefore, we need a mechanism to incentivize the users to hold their ALCA to have great illiquidity at the beginning to increase the odds of AliceNet success.
+
+### Goals
+<!--- What is goal this will accomplish -->
+
+Incentive illiquidity of ALCA at the beginning of the project. In other words, incentivize users to lock their ALCA for a certain amount of time until the price becomes more stable.
+
+### Non Goals
+<!--- What is not to be included with this -->
+
+Users don't lock their ALCA and dump them directly into the AMM markets.
+
+### Assumptions
+<!-- Conditions and resources that need to be present and accessible for the solution to work as described -->
+
+The ALCA and Public Staking contracts are deployed and available and a generous amount of ALCA is separated to be distributed to users that lock their ALCA.
```

Comment:
what do you think about changing this to a list 

- ALCA and Public Staking contracts are deloyed
- Conversion ratio of ALCA is 1 MadToken = 1.555555 ALCA

Reply:
Well, don't think it works here. The lockup system doesn't depend on the 1.555555 multiplier.